ARMmbed/meta-mbl
# Increase default log partition size

Some applications (e.g. edge-core) log a lot of information. Increase
the default log partition size to cope with this.

For: IOTMBL-2280 Update .wks files for new partition layout guidelines

**Jenkins links**
* http://jenkins.mbed-linux.arm.com/view/jh/job/jh-test/118/ (terminated; temporary(?) failure fetching meta-mbl).
* http://jenkins.mbed-linux.arm.com/view/jh/job/jh-test/119/ (Successful).

**Lava results**
* http://e120856-lin/results.php?image_version=custom&image_number=119&custom_version=jh-test&lava_query=&lava_owner=&failures=false&submitter=jonhai01&sort=by_job. Failures:
  * https://lava.mbedcloudtesting.com/scheduler/job/27922 (pico7 Wi-Fi failures as usual).

**Manual testing**
Check on warp7 that the log partition size is about right:
```
root@mbed-linux-os-9345:~# df -h /dev/disk/by-label/log 
Filesystem                Size      Used Available Use% Mounted on
/dev/mmcblk1p9          122.0M      1.6M    111.4M   1% /var/log
root@mbed-linux-os-9345:~# fdisk -l /dev/mmcblk1p9
Disk /dev/mmcblk1p9: 128 MiB, 134217728 bytes, 262144 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
root@mbed-linux-os-9345:~#
```
